x-pack/plugins/security_solution/public/common/components/integrations_popover/index.tsx

```diff
@@ -50,12 +51,13 @@ const IntegrationsPopoverComponent = ({ integrations }: IntegrationsPopoverProps
   const { data } = useInstalledIntegrations({ packages: [] });
   const basePath = useBasePath();
 
-  const allInstalledIntegrations: RelatedIntegrationArray = data ?? [];
+  const allInstalledIntegrations: InstalledIntegrationArray = data?.installed_integrations ?? [];
```

Comment:
Would be great if `useInstalledIntegrations` hook was doing this logic and always returning an array.

Reply:
Addressed in https://github.com/elastic/kibana/pull/133050 to return `data?.installed_integrations ?? []`, though note that the return type `UseQueryResult<InstalledIntegrationArray>` can still be undefined, so doesn't change the need for the downstream null checks